frontend/src/store/eam-group-configurate/actions.ts

```diff
@@ -0,0 +1,40 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import {
+  AsyncThunkConfig,
+  EAMGroupConfigurateRequestDto,
+  EAMGroupCreateRequestDto,
+  EAMGroupCreateResponseDto,
+  EAMWorkerGetAllResponseDto,
+  EAMWorkerGetByTenantRequestParamsDto,
+} from 'common/types/types';
+import { ActionType } from './common';
+
+const create = createAsyncThunk<
+  EAMGroupCreateResponseDto,
+  EAMGroupConfigurateRequestDto,
+  AsyncThunkConfig
+>(ActionType.CREATE, async (registerPayload, { getState, extra }) => {
+  const { eamApi } = extra;
+
+  const { app } = getState();
+  const { tenant } = app;
+
+  const request: EAMGroupCreateRequestDto = {
+    name: registerPayload.name,
+    tenantId: tenant?.id ?? '',
+    workersIds: registerPayload.workersIds ?? [],
```

Comment:
```suggestion
    workersIds: registerPayload.workersIds,
```
it seems like we always have an array